[2022-05-03 18:20:41,938] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2022-05-03T18:20:32.107320+00:00 [queued]>
[2022-05-03 18:20:41,954] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2022-05-03T18:20:32.107320+00:00 [queued]>
[2022-05-03 18:20:41,955] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-05-03 18:20:41,955] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-05-03 18:20:41,955] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-05-03 18:20:41,976] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_4> on 2022-05-03T18:20:32.107320+00:00
[2022-05-03 18:20:41,978] {standard_task_runner.py:52} INFO - Started process 96008 to run task
[2022-05-03 18:20:41,985] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', '2022-05-03T18:20:32.107320+00:00', '--job-id', '28', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp4dpw6qkk', '--error-file', '/tmp/tmpw_wh4z57']
[2022-05-03 18:20:41,986] {standard_task_runner.py:77} INFO - Job 28: Subtask task_4
[2022-05-03 18:20:42,070] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_4 2022-05-03T18:20:32.107320+00:00 [running]> on host airflowvm
[2022-05-03 18:20:42,197] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2022-05-03T18:20:32.107320+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-03T18:20:32.107320+00:00
[2022-05-03 18:20:42,197] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-05-03 18:20:42,198] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-05-03 18:20:42,204] {subprocess.py:75} INFO - Output:
[2022-05-03 18:20:45,207] {subprocess.py:83} INFO - Command exited with return code 0
[2022-05-03 18:20:45,291] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20220503T182032, start_date=20220503T182041, end_date=20220503T182045
[2022-05-03 18:20:45,321] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-05-03 18:20:45,355] {local_task_job.py:151} INFO - Task exited with return code 0
